docs/proposals/volume-health-monitoring.md

```diff
@@ -0,0 +1,266 @@
+# Proposal: Supporting for CSI volume health feature 
+
+Last updated: 24.12.2021
+
+## Abstract
+
+This proposal contains approaches for supporting CSI volume health feature, using Kubernetes volume health monitor mechanism. 
+
+## Background
+
+Currently, Baremetal CSI does not support any general conventional way to monitor Persistent Volumes after they are provisioned in Kubernetes.
```

Comment:
It does. Health and Missing operational status of PVs are exposed through the Volumes CRD

Reply:
Yes, and a bit later this was described, but it is not the standardized approach, which is native to k8s mechanisms (e.g. events, pvc, pods). The client/customer should know about CSI's own CRD (e.g. it's how implemented DR SP on operator)